src/utils/db/car_stream.rs

```diff
@@ -329,6 +329,12 @@ async fn read_v1_header<ReaderT: AsyncRead + Unpin>(
     Some(header)
 }
 
+pub fn uvi_bytes() -> UviBytes {
+    let mut decoder = UviBytes::default();
+    decoder.set_max_len(usize::MAX);
```

Comment:
Is this reasoanble? Shouldn't we still put some bounds on it, like at most 4 GiB?